flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CalcITCase.scala

```diff
@@ -548,4 +548,29 @@ class CalcITCase extends StreamingTestBase {
     TestBaseUtils.compareResultAsText(result, "42")
   }
 
+  @Test
+  def testStatementSetInsertUsingTableDescriptor(): Unit = {
```

Comment:
Move to `TableSinkITCase` or better just access the `StatementSet` if possible and avoid a Flink execution.